File: Baekjoon/Baekjoon09656.py
# Baekjoon09656.py
n = int(input())

is_win = [ True, False, True, False ]
for i in range(4, n + 1):
	is_win.append(False if (is_win[i - 1] and is_win[i - 3]) else True)

print("SK" if is_win[n] else "CY")

File: Baekjoon/Baekjoon09656.cpp
// Baekjoon09656.cpp
#include <iostream>

using namespace std;

int main(void) {
	int n;
	cin >> n;

	bool isWin[1'001];
	isWin[1] = false;
	isWin[2] = true;
	isWin[3] = false;

	for (int i = 4; i <= n; i++) {
		if (isWin[i - 1] && isWin[i - 3])
			isWin[i] = false;
		else
			isWin[i] = true;
	}

	cout << (isWin[n] ? "SK" : "CY") << '\n';

	return 0;
}